lib/pangea/widgets/chat/overlay_footer.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:fluffychat/config/themes.dart';
+import 'package:fluffychat/pages/chat/chat.dart';
+import 'package:fluffychat/pages/chat/chat_input_row.dart';
+import 'package:fluffychat/pages/chat/reactions_picker.dart';
+import 'package:flutter/material.dart';
+
+enum _EventContextAction { info, report }
```

Comment:
This enum is not references in this file and if already defined in chat_view.dart (and in overlay_header.dart) and can be removed.